packages/react-components/stories/title-bar.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import TitleBar from '../lib/title-bar';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import IconButton from '@material-ui/core/IconButton';
+
+export default {
+  title: 'Title Bar',
+  component: TitleBar,
+} as Meta;
+
+export const ExampleTitleBar: Story = (args) => {
```

Comment:
Would be good to make it support storybook controls and actions, see the tasks stories for some examples. Mainly, you need to let `{...args}` take care of the props, and not pass fixed props like `tabNames={tabNames}` explicitly. Some things like `logoPath` can be fixed, but try to make some of the props configurable from storybook.